cpp/ps0000/ps0000_sl.cpp

```diff
@@ -1,15 +1,16 @@
-#include <iostream> // importing libraries should be started with # not @ and name is iostream only
+#include <iostream> // importing libraries should be started with # not @ and name is iostream only 2/2
 
-// it's main not Main
+// it's main not Main 1/1
 int main() { 
 
     // it's << not >> we are printing data to the console not getting data
-    // add std::endl; to end the line and add a new line (break)
+    // add std::endl; to end the line and add a new line (break) 4/4
     std::cout << "Hello, World!" << std::endl;
 
     // same message with escape \n
     std::cout << "Hello, World!\n";
 
-    // main function runs your whole code return 0;
+    // main function runs your whole code return 0; 1/1
     return 0;
-}
\ No newline at end of file
+}
+// great work 😎 11/10
```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit
+1 bonus mark for your comments